src/pages/home/report/ReportActionItem.js

```diff
@@ -584,12 +584,7 @@ function ReportActionItem(props) {
                             draftMessage={props.draftMessage}
                             isChronosReport={ReportUtils.chatIncludesChronos(originalReport)}
                         />
-                        <View
-                            style={StyleUtils.getReportActionItemStyle(
-                                hovered || isWhisper || isContextMenuActive || props.draftMessage,
-                                (props.network.isOffline && props.action.isLoading) || props.action.error,
```

Comment:
There's also the case when action has error. When error, opacity should be 0.5.
@cubuspl42 please add this error case to Tests step as well.

Reply:
For a few seconds I was really confused, but now I get it; I just blindly removed the `isLoading` parameter, when `isLoading` was abused when there was an error. Thanks for spotting this.

Reply:
@situchan  Actually, maybe a silly question, but how to trigger an error so `action.error` is truthy?

Reply:
I am also not sure.
Please try repro step in https://github.com/Expensify/App/issues/27315.

If we're not clear, let's keep that code part. And only remove `isLoading` conditions.

Reply:
@bernhardoj I had a feeling that I've discussed this before, but this PR is so old that I already forgot the details...

@situchan If our conclusion from the other discussion looks good for you, we can leave this as-is. I'll also use the repro steps you linked to double confirm.

Reply:
![image](https://github.com/Expensify/App/assets/2590174/9f27348f-c554-4aa4-b488-35a33cc6e57e)

(with the code as-is)

Reply:
@situchan Please let me know if we can resolve this thread without code changes

Reply:
I think we can resolve